common/models/wordbreakers/tools/data-compiler/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "../../../tsconfig.kmw-worker-base.json",
+
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "../../build/obj/data-compiler",
```

Comment:
Would it be better if the tool outputs were separated out from the deployable outputs? e.g. `build/tools/data-compiler`?

Reply:
I've added a new commit to address this.